ee/dtr/admin/install/index.md

```diff
@@ -64,14 +64,14 @@ DTR and UCP must not be installed on the same node, due to the potential for res
 
     * To install a specific version of DTR, replace `{{ page.dtr_version }}` with the  desired version in the [installation command](#step-3-install-dtr). (All DTR versions can be found on the [DTR release notes](/ee/dtr/release-notes/) page.)
 
-    * As DTR is deployed with self-signed certificates by default, UCP might not be able to pull images it. To automatically configure UCP to trust DTR, use the `--dtr-external-url <dtr-domain>:<port>` optional flag during installation, or during a reconfiguration.
+    * As DTR is deployed with self-signed certificates by default, UCP might not be able to pull images from it. To automatically configure UCP to trust DTR, use the `--dtr-external-url <dtr-domain>:<port>` optional flag during installation, or during a reconfiguration.
 
     * With DTR 2.7, it is possible to [enable browser authentication via client
     certificates](/ee/enable-client-certificate-authentication/) at the time of install, and thus remove the need to enter user credentials.
 
 5. Verify that DTR is installed, either by navigating to `https://<ucp-fqdn>/manage/settings/dtr`, or by navigating in the UCP web UI to **Admin Settings > Docker Trusted Registry**.
 
-    Under the hood, UCP modifies `/etc/docker/certs.d` for each host and adds DTR's CA certificate. UCP can then pull images from DTR because the Docker Engine for each node in the UCP swarm has been configured to trust DTR.
+    Note that UCP modifies `/etc/docker/certs.d` for each host and adds the DTR CA certificate. UCP is thus able to pull images from DTR, because the Docker Engine for each node in the UCP swarm is configured to trust DTR.
 
```

Comment:
Can we remove the "note that" and just say "UCP modifies...".

Reply:
Done.